.github/workflows/release.yml

```
@@ -9,60 +9,227 @@ permissions:
   build:
-    runs-on: ubuntu-latest
-    outputs:
-      release_dir: ${{ steps.set_release_dir.outputs.release_dir }}
+    name: Build and Verify Release Binaries
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
     steps:
       - name: Checkout Code
         uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)